content/ngf/how-to/data-plane-configuration.md

```diff
@@ -97,12 +219,54 @@ parametersRef:
     group: gateway.nginx.org
     kind: NginxProxy
     name: ngf-proxy-config
+    namespace: default
 ```
 
 After updating, you can check the status of the GatewayClass to see if the configuration is valid:
 
 ```shell
-kubectl describe gatewayclasses nginx
+kubectl describe gatewayclasses <gateway-class-name>
```

Comment:
Should we just leave this since it's the default name and will be easier for users to copy?

Reply:
Sure